fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/plans/RecordQueryCoveringIndexPlan.java

```diff
@@ -190,7 +190,7 @@ public int planHash() {
     @Nonnull
     @Override
     @API(API.Status.EXPERIMENTAL)
-    public Iterator<? extends ExpressionRef<? extends RelationalExpression>> getPlannerExpressionChildren() {
-        return Iterators.singletonIterator(indexPlan);
+    public List<? extends Quantifier> getQuantifiers() {
```

Comment:
Should this return the indexPlan?

Reply:
No, it shouldn't. We decided that `RecordQueryCoveringIndexPlan` is basically a "wrapper" around a `RecordQueryIndexPlan` that happens to use composition rather than inheritance. The child isn't really a child in the relational expression sense.

Reply:
done